scripts/securedrop-admin.py

```diff
@@ -79,6 +79,22 @@ def validate_config(path):
         raise SDAdminException("Error while validating configuration")
 
 
+def refresh_salt():
+    """
+    Cleans the Salt cache and synchronizes Salt to ensure we are applying states
+    from the currently installed version
+    """
+    try:
+        subprocess.check_call(["sudo", "rm", "-rf", "/var/cache/salt"])
```

Comment:
It's worth noting there's also `qubesctl saltutil.clear_cache`, which does leave some scaffolding in place, but is a bit less drastic. The removal is recommended by the `qubes-mgtm-salt` README [here](https://github.com/QubesOS/qubes-mgmt-salt/blob/master/README.rst).

Reply:
Comparing the state after running each command:

* `rm -rf /var/cache/salt` removes _everything_, as you might expect
* `qubesctl saltutil.clear_cache` removes _only files_, but leaves the directories. that sounds acceptable to me, but given that the qubes docs are still technically recommending an rm, let's do that. We also have a recent report that these commands did indeed resolve an issue https://github.com/freedomofpress/securedrop-workstation/issues/527#issuecomment-611709172